cypress/src/tests/e2eTests/events/add-to-cart.spec.js

```diff
@@ -0,0 +1,52 @@
+import { products } from "../../../fixtures";
+import { expectsEventWithContext } from "../../../assertions";
+/**
+ * https://github.com/adobe/commerce-events/blob/main/examples/events/initiate-checkout.md
```

Comment:
Make sure you change this to the add-to-cart doc :)

Reply:
done!